overrides/home.html

```diff
@@ -320,11 +320,11 @@ <h1>NHS England Data Science</h1>
         </a>
 
         <a
-        href="what_is_data_science/How you can learn Data Science/"
-        title="See our work"
+        href="PhDInterns/"
+        title="PhD Intenrship Scheme"
         class="md-button"
         >
-        How can I learn data science?
+        See our PhD Internship Scheme?
```

Comment:
does this need a question mark?

Reply:
also whilst im here, any thoughts on a replacement image for the landing page?

Reply:
Nope - removed.  

Will think of replacement image for landing page separately